core/trino-grammar/src/main/antlr4/io/trino/grammar/sql/SqlBase.g4

```diff
@@ -916,6 +916,16 @@ sqlStatementList
 
 privilege
     : CREATE | SELECT | DELETE | INSERT | UPDATE
+    | IDENTIFIER
```

Comment:
Privilege formatting changed.  I picked `IDENTIFIER` because I thought we would be unlikely to need or want quoted or digit identifiers.  But we likely want to allow `nonReserverd` tokens to be privileges, so `identifier` is the better choice.  Changed.